java/api/src/main/java/org/ray/api/id/ActorId.java

```diff
@@ -0,0 +1,50 @@
+package org.ray.api.id;
+
+import java.io.Serializable;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Arrays;
+import java.util.Random;
+
+public class ActorId extends BaseId implements Serializable {
+  private static final int UNIQUE_BYTES_LENGTH = 4;
+
+  public static final int LENGTH = UNIQUE_BYTES_LENGTH + JobId.LENGTH;
+
+  public static final ActorId NIL = nil();
+
+  private ActorId(byte[] id) {
+    super(id);
+  }
+
+  public static ActorId fromByteBuffer(ByteBuffer bb) {
+    return new ActorId(byteBuffer2Bytes(bb));
+  }
+
+  public static ActorId generateActorId(JobId jobId) {
```

Comment:
One conventional and shorter name for factory methods is `ActorId.of(jobId)`. Current name is okay as well.